Template/Tag/ZendeskChatTag.web.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var zendeskChatId = parameters.get("zendeskChatId");
+            window.$zopim || (function () {
```

Comment:
actually, maybe we could add a check to execute it only if a zendeskChatId exists: `if (! zendeskChatId) {return;}`

Reply:
While a check never hurts, I can't think of a way it could be undefined if it is required.